containers/home.tsx

```diff
@@ -1,66 +1,89 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import VideoCardList from '../components/videoCardList/videoCardList';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [currentVideoIndex, setCurrentVideoIndex] = useState(0);
+  const [videosToShow, setVideosToShow] = useState([]);
+
+  const cards = [
+    { title: 'Course 1', alt: 'img1', src: '1' },
+    { title: 'Course 2', alt: 'img2', src: '2' },
+    { title: 'Course 3', alt: 'img3', src: '3' },
+    { title: 'Course 4', alt: 'img4', src: '4' },
+    { title: 'Course 5', alt: 'img5', src: '5' },
+    { title: 'Course 6', alt: 'img6', src: '6' },
+  ];
+
+  useEffect(() => {
+    const videosToShow = [];
+
+    for (let i = currentVideoIndex; i <= currentVideoIndex + 3; i++) {
+      videosToShow.push(cards[i]);
+    }
+
+    setVideosToShow(videosToShow);
+  }, [currentVideoIndex]);
+
+  const arrowNext = () => {
+    if (currentVideoIndex < 2) {
+      setCurrentVideoIndex(currentVideoIndex + 1);
+    }
+  };
+
+  const arrowPrev = () => {
+    if (currentVideoIndex > 0) {
+      setCurrentVideoIndex(currentVideoIndex - 1);
+    }
+  };
```

Comment:
I just noticed that these are functions passed to the buttons, maybe we can change their names to clarify that. Something like `setNextVideoIndex` and `setPrevVideoIndex`. What do you think?